.github/workflows/deploy-api-client.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy api and client
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'client/**'
+      - 'api/**'
+      - '.github/workflows/deploy-api-client.yml'
+      - 'docker-compose.yaml'
+  workflow_dispatch:
+
+env:
+  VITE_SESSION_SECRET: ${{ secrets.VITE_SESSION_SECRET }}
+  DUCKDB_POOL_SIZE: 5
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    - uses: alex-ac/github-action-ssh-docker-compose@master
```

Comment:
As this is a critical action I would pin this to a specific hash as a tag can be force pushed. To keep it up to date we can [add dependabot ](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file) to keep security fixes coming.

Reply:
Hey @assignUser we'll address this on a different PR, thanks